sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -1785,10 +1941,88 @@ def df_getitem_iat_tuple_impl(self, idx):
 
         raise TypingError('Operator getitem(). The index must be a row and literal column. Given: {}'.format(idx))
 
+    if accessor == 'iloc':
+        if isinstance(idx, types.SliceType):
+            return gen_df_getitem_iloc_slice_impl(self.dataframe, idx)
+
+        if (
+            isinstance(idx, (types.List, types.Array)) and
+            isinstance(idx.dtype, (types.Boolean, bool))
+        ):
+            return gen_df_getitem_iloc_list_bool_impl(self.dataframe, idx)
+
+        if isinstance(idx, types.List):
+            return gen_df_getitem_iloc_list_impl(self.dataframe, idx)
+
+        if isinstance(idx, types.Integer):
+            return gen_df_getitem_iloc_int_impl(self.dataframe, idx)
+
+        if isinstance(idx, (types.Tuple, types.UniTuple)):
+            def df_getitem_tuple_iat_impl(self, idx):
+                return self._dataframe.iat[idx]
+
+            return df_getitem_tuple_iat_impl
+
     raise TypingError('Operator getitem(). Unknown accessor. Only "loc", "iloc", "at", "iat" are supported.\
                       Given: {}'.format(accessor))
 
 
+@sdc_overload_attribute(DataFrameType, 'iloc')
+def sdc_pandas_dataframe_iloc(self):
+    """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.DataFrame.iloc
+
+    Limitations
+    -----------
+    - Parameter 'name' in new DataFrame can be String only
+    - Column can be literal value only, in DataFrame.iloc[row, column]
+    - Iloc works with basic cases only: an integer, a list or array of integers,
+        a slice object with ints, a boolean array
+
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/dataframe/dataframe_iloc.py
+       :language: python
+       :lines: 27-
```

Comment:
```suggestion
       :lines: 36-
```

Please check the documentation visually.